sqld/src/connection/config.rs

```diff
@@ -23,6 +23,24 @@ pub struct DatabaseConfig {
     /// The reason why operations are blocked. This will be included in [`Error::Blocked`].
     #[serde(default)]
     pub block_reason: Option<String>,
+    /// maximum db size (in pages)
+    #[serde(default = "default_max_size")]
+    pub max_db_pages: u64,
+}
+
+const fn default_max_size() -> u64 {
+    bytesize::ByteSize::pb(1000).as_u64() / 4096
```

Comment:
nitpick: `LIBSQL_PAGE_SIZE` instead of 4096?

Reply:
thanks!